src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Tippy from '@tippyjs/react';
+import 'tippy.js/dist/tippy.css';
+
+type TootipProps = {
+  children: React.ReactElement;
+  tooltipContent: React.ReactElement;
+};
+export const Tooltip: React.FC<TootipProps> = ({ tooltipContent, children, ...rest }) => (
```

Comment:
would 'tooltipContent' be better ar 'content'? I'm not sure so just asking the question, the thought is, it is in the tooltip, so it doesn't need to say 'tooltip' again?